.vib/grafana-loki/cypress/cypress/integration/grafana_loki_spec.js

```diff
@@ -0,0 +1,71 @@
+/// <reference types="cypress" />
+import body from '../fixtures/logs-body.json';
+
+it('shows build info endpoint', () => {
+  cy.request({
+    method: 'GET',
+    url: '/loki/api/v1/status/buildinfo',
+    form: true,
+  }).as('buildInfo');
+  cy.get('@buildInfo').should((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body).to.include.all.keys(
+      'version',
+      'revision',
+      'branch',
+      'buildUser',
+      'goVersion'
+    );
+  });
+});
+
+it('allows Loki to get Grafana logs', () => {
+  cy.request({
+    method: 'GET',
+    url: 'loki/api/v1/query?query={app="grafana-loki"}',
+    form: true,
+  }).as('grafanaLogs');
+  cy.get('@grafanaLogs').should((response) => {
+    expect(response.status).to.eq(200);
```

Comment:
Chances are this is returning a JSON file. Then, maybe it is worth to check the content-type is set to JSON as well:

```
expect(response.headers['content-type']).to.eq('application/json');
```

Reply:
Thanks for the suggestion! I've added the line below.